<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatClientWidget</class>
 <widget class="QWidget" name="ChatClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Veyon Chat - Student</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/chat.qrc">
    <normaloff>:/chat/icons/chat.png</normaloff>:/chat/icons/chat.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="chatLabel">
     <property name="text">
      <string>Chat with Teacher</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="chatDisplay">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="messagePanel">
     <layout class="QHBoxLayout" name="messageLayout">
      <item>
       <widget class="QLineEdit" name="messageInput">
        <property name="placeholderText">
         <string>Type your message here...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendButton">
        <property name="text">
         <string>Send</string>
        </property>
        <property name="toolTip">
         <string>Send message (Ctrl+Enter)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="statusPanel">
     <layout class="QHBoxLayout" name="statusLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Ready</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/chat.qrc"/>
 </resources>
 <connections/>
</ui>
